development/bin/ensure_release_notes.py

```diff
@@ -0,0 +1,93 @@
+"""Ensure that release notes exist for a given version.
+
+This script will do the following:
+    Ensure a release notes file exists at `docs/admin/release_notes/version_{version}.md`.
+    Ensure the `mkdocs.yml` file is updated to add the release notes file to the navigation.
+    Ensure the `pyproject.toml` `tool.towncrier.filename` is updated to reference the release notes file.
+
+It shouldn't be necessary to run this file manually. It is automatically called by `invoke generate-release-notes`.
+
+Example:
+    $ python ensure_release_notes.py --version '1.0.0'
+"""
+
+import argparse
+
+try:
+    import tomllib
+except ImportError:
+    import tomli as tomllib
+
+from pathlib import Path
+
+
+def release_notes_pyproject_toml(version):
+    """Update the pyproject.toml file to set the towncrier filename for the given version."""
+    pyproject_file = Path(__file__).parent.parent.parent / "pyproject.toml"
+    pyproject_content = pyproject_file.read_text()
+    pyproject_data = tomllib.loads(pyproject_content)
+
+    # Update the towncrier filename
+    if pyproject_data["tool"]["towncrier"].get("filename", "") != f"docs/admin/release_notes/version_{version}.md":
+        pyproject_data["tool"]["towncrier"]["filename"] = f"docs/admin/release_notes/version_{version}.md"
+
+        # Write back the updated content to pyproject.toml
```

Comment:
```suggestion
        # Write back the updated content to pyproject.toml
        # tomllib is not used to write the file because it is not roundtrippable
```